http-clients/apache-client/src/main/java/software/amazon/awssdk/http/apache/ApacheHttpClient.java

```diff
@@ -133,6 +134,15 @@ public static Builder builder() {
         return new DefaultBuilder();
     }
 
+    /**
+     * Create a {@link HttpConnection} client with the default properties
+     *
+     * @return an {@link ApacheHttpClient}
+     */
+    public static SdkHttpClient create() {
+        return new DefaultBuilder().build();
```

Comment:
Query : Just wondering if we need to create it with Default value as below
```
    public static SdkHttpClient create() {
        return new DefaultBuilder().buildWithDefaults(AttributeMap.empty());
    }
```
This will end up in initializing the values with [GLOBAL_HTTP_DEFAULTS](https://github.com/aws/aws-sdk-java-v2/blob/cdfb7d3f6a7dafb37be84f6e95e49feb0d0af273/http-client-spi/src/main/java/software/amazon/awssdk/http/SdkHttpConfigurationOption.java#L130)

else we end up with Default build as [SdkHttpClient Builder](https://github.com/aws/aws-sdk-java-v2/blob/cdfb7d3f6a7dafb37be84f6e95e49feb0d0af273/http-client-spi/src/main/java/software/amazon/awssdk/http/SdkHttpClient.java#L67-L69)

When we create should it   created with Empty values or with GLOBAL_HTTP_DEFAULTS ?